com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/TessellationShare.hlsl

```diff
@@ -160,6 +160,11 @@ PackedVaryingsToPS Domain(TessellationFactors tessFactors, const OutputPatch<Pac
     varying.vmesh.positionRWS = PhongTessellation(  varying.vmesh.positionRWS,
                                                     p0, p1, p2, n0, n1, n2,
                                                     baryCoords, _TessellationShapeFactor);
+
+#ifdef VARYINGS_NEED_POSITIONPREDISPLACEMENT_WS
+    varying.vmesh.positionPredisplacementRWS = varying.vmesh.positionRWS;
```

Comment:
be careful you are in the _TESSELLATION_PHONG bloc, you should be outside this bloc

Reply:
OK, I'll move outside of the block.